src/components/Event/Event.st.css

```diff
@@ -42,10 +42,11 @@
 }
 
 .root {
-    -st-states: selected, fullday, rtl;
```

Comment:
it was decided that the rtl feature will not be implemented in the tpa ? because we need it

Reply:
I have to admit that I don't remember why I removed it 🤔 
But I believe it might not be necessary to use the `rtl` state, since flex automatically adjusts itself to `dir="rtl"`
I'll have to check of course 👍